lib/ui/views/splash/widgets/curved_clipper.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class CurvedShapeClipper extends CustomClipper<Path> {
+  @override
+  Path getClip(Size size) {
+    final path = Path();
+
+    print('device height ${size.height}');
+    print('device width ${size.width}');
```

Comment:
same as above

Reply:
Fixed using the Logger class , wrt to the getLogger , does it run only in debug mode only ? 
or also in production as well...

Reply:
debug mode only